Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -27,6 +27,29 @@ namespace Babylon
 {
     namespace
     {
+        constexpr auto YFlipVertexShader = R"(
+            precision highp float;
+            in vec2 position;
+            out vec2 vUV;
+            void main(void)
+            {
+                gl_Position = vec4(position, 0.5, 1.0);
+                vUV = position * 0.5 + 0.5;
+            }
+        )";
+
+        constexpr auto YFlipFragmentShader = R"(
+            precision highp float;
+            in vec2 vUV;
+            uniform sampler2D backBuffer;
+            out vec4 glFragColor;
+            void main(void)
+            {
+                vec3 color = texture(backBuffer, vUV).xyz;
+                glFragColor = vec4(color, 1.0);
+            }
+        )";
```

Comment:
Maybe I'm being dense, but I don't see how this shader flips the y axis. How does this work?

Reply:
The shader itself doesn't flip. It only copies. UV are handled so texture tap and pixel output make the flip.